hudi-hadoop-common/src/test/java/org/apache/hudi/common/fs/TestStorageSchemes.java

```diff
@@ -39,6 +39,31 @@ public void testStorageSchemes() {
     assertTrue(StorageSchemes.isSchemeSupported("afs"));
     assertFalse(StorageSchemes.isSchemeSupported("s2"));
 
+    assertTrue(StorageSchemes.getStorageLockImplementationIfExists("s3").isPresent());
```

Comment:
might be simpler to iterator through all values in the ENUM and do `if` or `else` within to either do assertTrue or assertFalse